lib/code_climate/test_reporter/formatter.rb

```diff
@@ -98,7 +99,13 @@ def round(numeric, precision)
         Float(numeric).round(precision)
       end
 
-      private
+      def write_to_file?
+        return true if tddium? || ENV["CODECLIMATE_TO_FILE"]
+        if ENV["TO_FILE"]
+          warn "TO_FILE is deprecated, use CODECLIMATE_TO_FILE"
```

Comment:
warn is in kernel and basically is a `$stderr.puts`

On Sat, Apr 18, 2015 at 7:28 AM, Noah Davis <notifications@github.com>
wrote:

> In lib/code_climate/test_reporter/formatter.rb
> <https://github.com/codeclimate/ruby-test-reporter/pull/84#discussion_r28644409>
> :
>
> > @@ -98,7 +99,13 @@ def round(numeric, precision)
> >          Float(numeric).round(precision)
> >        end
> >
> > -      private
> > +      def write_to_file?
> > +        return true if tddium? || ENV["CODECLIMATE_TO_FILE"]
> > +        if ENV["TO_FILE"]
> > +          warn "TO_FILE is deprecated, use CODECLIMATE_TO_FILE"
>
> Where is warn defined?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeclimate/ruby-test-reporter/pull/84/files#r28644409>
> .
>